platform-sdk/swirlds-common/src/main/java/com/swirlds/common/config/sources/ConfigSourceOrdinalConstants.java

```diff
@@ -47,6 +47,12 @@ public final class ConfigSourceOrdinalConstants {
 
     @Deprecated(forRemoval = true)
     public static final int LEGACY_PROPERTY_FILE_ORDINAL_FOR_CONFIG = LEGACY_PROPERTY_FILE_ORDINAL + 20;
+
+    /**
+     * Ordinal for configuration passed to the platform builder via withConfig().
+     */
+    public static final int PLATFORM_BUILDER_ORDINAL = 50;
```

Comment:
Please do not add platform dependent content to common. Can you move that const to platform-core?

Reply:
removed